plugins/azure/fix_plugin_azure/collector.py

```diff
@@ -224,8 +225,11 @@ def collect_with(self, builder: GraphBuilder, locations: Dict[str, BaseRegion])
         regional_resources = [r for r in subscription_resources if resource_with_params(r, "location")]
         global_resources = list(set(subscription_resources) - set(regional_resources))
         self.collect_resource_list("subscription", builder, global_resources)
+        processed_locations: Set[str] = set()
         for location in locations.values():
-            self.collect_resource_list(location.safe_name, builder.with_location(location), regional_resources)
+            if location.safe_name not in processed_locations:
+                self.collect_resource_list(location.safe_name, builder.with_location(location), regional_resources)
+                processed_locations.add(location.safe_name)
```

Comment:
why do we need this? do we have multiple location with the same name?

Reply:
some resources have location in display format. `westeu` can be `West Eu` in some resources. but there are not many such resources